data/json/monstergroups/wilderness.json

```diff
@@ -2861,7 +2869,11 @@
         "starts": 360,
         "conditions": [ "SPRING", "SUMMER", "AUTUMN" ]
       },
-      { "monster": "mon_mutant_experimental", "freq": 2, "cost_multiplier": 6 }
+      { "monster": "mon_mutant_experimental", "freq": 2, "cost_multiplier": 6 },
+      {"monster": "mon_mantis_small", "freq": 2, "cost_multiplier": 8},
+      {"monster": "mon_aphid", "freq": 10, "cost_multiplier": 0},
+      { "monster": "mon_antlion_giant", "freq": 4, "cost_multiplier": 10 },
+      { "monster": "mon_strider_small", "freq": 15, "cost_multiplier": 3, "pack_size": [ 4, 8 ] }
```

Comment:
There really isn't enough open water in our swamps for striders to make sensem especially in those pack sizes - crayfish have the same problem, but I haven't gotten around to moving them out.
```suggestion
```